lib/almodovar.rb

```diff
@@ -16,14 +16,27 @@
 require 'almodovar/to_xml'
 
 module Almodovar
-
   class << self
+    DEFAULT_SEND_TIMEOUT = 30
+    DEFAULT_CONNECT_TIMEOUT = 30
+    DEFAULT_RECEIVE_TIMEOUT = 30
+
     def default_options
-      @default_options ||= {
-        :timeout => 120,
-        :connect_timeout => 30,
-        :user_agent => "Almodovar/#{Almodovar::VERSION}"
+      default = {
+        send_timeout: DEFAULT_SEND_TIMEOUT,
```

Comment:
@maxcosworth IMO the 3 timeout are different and we should provide an API to set them.

This [link](https://github.com/ankane/the-ultimate-guide-to-ruby-timeouts) is in the original issue and you can see the different timeouts even if we use the library (httpclient).